tests/copy.bats

```diff
@@ -446,24 +446,35 @@ stuff/mystuff"
 
 @test "copy-preserving-extended-attributes" {
   createrandom ${TESTDIR}/randomfile
-  image="quay.io/libpod/fedora-minimal:34"
+  image="registry.fedoraproject.org/fedora-minimal:35"
```

Comment:
A thought for later.  Should we define `${FEDORA_MINIMAL}` or some such in helpers.bat?  That way if we need to bump from 35 to 36, we only have one place to do it.

Reply:
This could as likely have left the version tag off of the image spec, since the test doesn't care about it as much, but don't let me stop you.

Reply:
The problem with omitting the version spec is that the next (or next-next) version will introduce some sort of breakage that will cause us to scramble in a panic. . Example: https://github.com/containers/podman/pull/12343

Reply:
See also #3640. If there is a critical need to use f35 here, can we push a copy to quay instead?

Reply:
It isn't critical that it be Fedora or Fedora-based, more that it have working `setcap`, `getfattr`, and `setfattr` commands.  I'll add comment there to try to clarify that.  If you've another image in mind that would work better, I'm happy to switch to using it.

Reply:
My question was, is `fedora-minimal:34` not working? If it no longer works, can we push a new image to quay? If it still works, can we keep it?

Reply:
I guess it will.  I switched to 35 mainly because I know 34 is going EOL in a little over a month.  Changing it back to 34.